forum/migration_helpers.py

```diff
@@ -236,9 +240,8 @@ def migrate_read_states(db: Database[dict[str, Any]], course_id: str) -> None:
     """Migrate read states from mongo to mysql."""
     users = db.users.find({"course_stats.course_id": course_id})
     for user_data in users:
-        try:
-            user = User.objects.get(id=int(user_data["_id"]))
-        except User.DoesNotExist:
+        user = User.objects.filter(id=int(user_data["_id"])).first()
+        if not user:
```

Comment:
How is this better?

Reply:
I just avoided `try-catch` as `.get()` throws a `DoesNotExist`  error but `filter().first()` doesn't.

Reply:
I see that you prefer to "ask permission not forgiveness" ;) 
https://stackoverflow.com/questions/12265451/ask-forgiveness-not-permission-explain
https://realpython.com/python-lbyl-vs-eafp/#the-pythonic-way-to-go-lbyl-or-eafp